andela/Ah-frontend-xmen
# #162725228 Article tags should be present in all relevant views

**What does this PR do?**
This PR enables users to view various article tags to enable easy filtering and navigation to articles of interest.

**Description of tasks completed**
- Create fetch all tags actions
- Setup a reducer 
- Build a container component
- Build a presentational component
- Return a list of all tags for easy navigation

**How this should be manually tested**
- Fetch and check out this branch using `git fetch origin ft-view-tags-162725228 && git checkout ft-view-tags-162725228`
- Run `yarn install && yarn start`. 
- Go to the landing page, where you'll view all tags and scroll through them. And to view tags of a single article, click on the article and scroll down to view its tags

**Related Pivotal Tracker Story**
 [#162725228](https://www.pivotaltracker.com/story/show/162725228)
